lib/prepare_security.js

```diff
@@ -16,31 +25,63 @@ export default class SecurityReleaseSteward {
     });
 
     const req = new Request(credentials);
-    const create = await cli.prompt(
+    const release = new PrepareSecurityRelease(req);
+    const releaseDate = await release.promptReleaseDate(cli);
+    const createIssue = await cli.prompt(
       'Create the Next Security Release issue?',
       { defaultAnswer: true });
-    if (create) {
-      const issue = new SecurityReleaseIssue(req);
-      const content = await issue.buildIssue(cli);
-      const data = await req.createIssue('Next Security Release', content, {
-        owner: 'nodejs-private',
-        repo: 'node-private'
-      });
-      if (data.html_url) {
-        cli.ok('Created: ' + data.html_url);
-      } else {
-        cli.error(data);
-      }
+    if (createIssue) {
+      // prepare the security release issue content and release date
+      const { content } = await release.buildIssue(releaseDate);
+
+      // create the security release issue
+      await release.createIssue(content, { cli });
```

Comment:
I think we could include a reference to the vulnerabilities.json PR in the main issue, so instead of creating the issue now, you do it after `createVulnerabilitiesJSON` and make it required instead of optional through a prompt.